packages/qr-code-generator/src/agent/QrCodeProvider.tsx

```diff
@@ -107,6 +107,9 @@ export class QrCodeProvider implements IAgentPlugin {
     }
     const scheme = (object.scheme ?? 'openid-credential-offer://').replace('://?', '').replace('://', '') + '://'
     const baseUri = object.baseUri ?? ''
+    if (object.scheme && baseUri.includes(object.scheme)) {
```

Comment:
This is not doing what I suggested

I am okay if they are the same.
I a not okay if they differ.

So a check on whether the scheme is part of it. If so ignore it. If there is a scheme and the base URI does not contain a scheme (check for ://) it should be appended. If both are present and they differ an error should be thrown

Reply:
Fixed!

Reply:
I think it's fixed now